services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/helpers/FlowHelper.groovy

```diff
@@ -17,9 +17,10 @@ class FlowHelper {
     def random = new Random()
     def allowedVlans = 101..4095
     def sdf = new SimpleDateFormat("ddMMMHHmmss", Locale.US)
+
     FlowPayload randomFlow(Switch srcSwitch, Switch dstSwitch) {
         new FlowPayload(sdf.format(new Date()), getFlowEndpoint(srcSwitch), getFlowEndpoint(dstSwitch), 500,
```

Comment:
It may be a good idea to use "autotest-flow-${sdf.format(new Date())}" so that autocreated flows could be deleted using a regex (sometime in future)

Reply:
That'd be good. but
1. Not the case of this PR
2. Please prefer underscores `_` over dashes `-` because it is easier to select it with double click during manual postman debug
3. We have an 'autotest flow' description

Reply:
Fix it in a separate patch.